doc/API-Friendica.md

```diff
@@ -0,0 +1,682 @@
+Delete the specified group of contacts; API call need to include the correct gid AND name of the group to be deleted.
+
+#### Parameters
+
+* `gid`: id of the group to be deleted
+* `name`: name of the group to be deleted
+
+#### Return values
+
+Array of:
+
+* `success`: true if successfully deleted
+* `gid`: gid of the deleted group
+* `name`: name of the deleted group
+* `status`: "deleted" if successfully deleted
+* `wrong users`: empty array
+
+---
+
+### GET api/friendica/notifications
+
+Return last 50 [Notifications](help/API-Entities#Notification) for the current user, ordered by date with unseen item on top.
+
+#### Parameters
+
+none
+
+### POST api/friendica/notifications/seen
+
+Set notification as seen.
+
+#### Parameters
+
+- `id`: id of the notification to set seen
+
+#### Return values
+
+If the note is linked to an item, returns an [Item](help/API-Entities#Item).
+
+Otherwise, a success status is returned:
+
+* `success` (json) | `<status>success</status>` (xml)
+
+---
+
+### GET api/friendica/photo
+
+Returns a [Photo](help/API-Entities#Photo).
+
+#### Parameters
+
+* `photo_id`: Resource id of a photo.
+* `scale`: (optional) scale value of the photo
+
+Returns data of a picture with the given resource.
+If 'scale' isn't provided, returned data include full url to each scale of the photo.
+If 'scale' is set, returned data include image data base64 encoded.
+
+possibile scale value are:
+
+* 0: original or max size by server settings
+* 1: image with or height at <= 640
+* 2: image with or height at <= 320
+* 3: thumbnail 160x160
+* 4: Profile image at 300x300
+* 5: Profile image at 80x80
+* 6: Profile image at 48x48
+
+An image used as profile image has only scale 4-6, other images only 0-3
+
+#### Return values
+
+json:
+
+```json
+	{
+		"id": "photo id"
+		"created": "date(YYYY-MM-DD HH:MM:SS)",
+		"edited": "date(YYYY-MM-DD HH:MM:SS)",
+		"title": "photo title",
+		"desc": "photo description",
+		"album": "album name",
+		"filename": "original file name",
+		"type": "mime type",
+		"height": "number",
+		"width": "number",
+		"profile": "1 if is profile photo",
+		"link": {
+			"<scale>": "url to image"
+			...
+		},
+		// if 'scale' is set
+		"datasize": "size in byte",
+		"data": "base64 encoded image data"
+	}
+```
+
+xml:
+
+```xml
+	<photo>
+		<id>photo id</id>
+		<created>date(YYYY-MM-DD HH:MM:SS)</created>
+		<edited>date(YYYY-MM-DD HH:MM:SS)</edited>
+		<title>photo title</title>
+		<desc>photo description</desc>
+		<album>album name</album>
+		<filename>original file name</filename>
+		<type>mime type</type>
+		<height>number</height>
+		<width>number</width>
+		<profile>1 if is profile photo</profile>
+		<links type="array">
+		<link type="mime type" scale="scale number" href="image url"/>
+			...
+		</links>
+	</photo>
+```
+
+### GET api/friendica/photos/list
+
+Returns the API user's [Photo List Items](help/API-Entities#Photo+List+Item).
+
+#### Return values
+
+json:
+
+```json
+	[
+		{
+			id: "resource_id",
```

Comment:
Isn't the key of a json a string too?

Reply:
It actually showed up as an error in my IDE!